.github/workflows/examples-efr32.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Build example - EFR32
+name: Build example - Silicon Labs
```

Comment:
this rename probably needs updates in 'required builds' for PRs since those are name based.

Is this needed? creates about 1 day worth of hickup in rebuilds.